apps/nodejs/extension/src/controllers/datasource/Transaction.ts

```diff
@@ -56,15 +56,17 @@ export type SwapTo = BaseTransaction["swapTo"];
 
 export const PoktShannonTransaction = BaseTransaction.extend({
   protocol: z.literal(SupportedProtocols.Cosmos),
-  fee: z.number(),
   memo: z.string().optional(),
   to: z.string().optional(),
+  fee: z.number().optional(),
   type: z
     .nativeEnum(CosmosTransactionTypes)
     .default(CosmosTransactionTypes.Send),
   transactionParams: z
     .object({
-      maxFeePerGas: z.number(),
+      gasPrice: z.number().optional(),
+      gas: z.union([z.literal("auto"), z.number()]).optional().default("auto"),
+      gasAdjustment: z.number().optional().default(1.1),
```

Comment:
I think you should left maxFeePerGas in case an user have a transaction saved with this option.

If that's the case, then they won't be able to export/import their vault.